docs/source/guides/user/chapters/plugins.rst

```diff
@@ -3,6 +3,15 @@ Understanding the plugin system
 
 Avocado has a plugin system that can be used to extended it in a clean way.
 
+.. note:: A large number of out-of-the-box Avocado features are
+  implemented as using the same plugin architecture available to
+  third-party extensions.
+
+  This guide considers "core", although still plugable, features those
```

Comment:
Suggestion: "This guide considers "core features", even though they're still "plugable", those available with an installation of Avocado by itself..."